blog/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class User(AbstractUser):
+    pass
+
+
+class Post(models.Model):
+    owner = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="posts")
+    title = models.CharField(max_length=255)
+    content = models.TextField()
+    created_time = models.DateTimeField(auto_now_add=True)
+
+
+class Commentary(models.Model):
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="commentaries")

```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Please, resolve this issue here and in the other places as well